src/main/webapp/index-cards/index-cards.js

```diff
@@ -45,12 +45,8 @@ export const DeleteAction = props => {
 }
 
 export const ShareAction = props => {
-  const { onShare } = props
-  return (
-    <IconButton onClick={onClick(onShare)}>
-      <ShareIcon />
-    </IconButton>
-  )
+  const { id, title, metacardType } = props
+  return <SharingModal id={id} title={title} metacardType={metacardType} />
```

Comment:
When we save a metacard, the metacardtype is required to be supplied by the front end otherwise it defaults to the default value for metacard type. This causes saving to break. I'm not sure if there's a better way to get the metacard type to the sharing component other than at this level.